exporter/awsemfexporter/emf_exporter.go

```diff
@@ -172,26 +172,11 @@ func (emf *emfExporter) Start(ctx context.Context, host component.Host) error {
 }
 
 func generateLogEventFromMetric(metric pdata.Metrics, config *Config) ([]*LogEvent, int, string) {
-	namespace := config.Namespace
-	rms := metric.ResourceMetrics()
-	cwMetricLists := []*CWMetrics{}
-	var cwm []*CWMetrics
 	var totalDroppedMetrics int
-
-	for i := 0; i < rms.Len(); i++ {
-		rm := rms.At(i)
-		if rm.IsNil() {
-			continue
-		}
-		cwm, totalDroppedMetrics = TranslateOtToCWMetric(&rm, config)
-		if len(cwm) > 0 && len(cwm[0].Measurements) > 0 {
-			namespace = cwm[0].Measurements[0].Namespace
-		}
-		// append all datapoint metrics in the request into CWMetric list
-		cwMetricLists = append(cwMetricLists, cwm...)
-	}
-
-	return TranslateCWMetricToEMF(cwMetricLists), totalDroppedMetrics, namespace
+	groupedMetricMap := make(map[string]*GroupedMetric)
+	totalDroppedMetrics = TranslateOtToGroupedMetric(metric, groupedMetricMap, config)
```

Comment:
Since we are not going to make it work end2end, I think you can implement the `TranslateOtToGroupedMetric` only and leave the remaining code unchanged. I think that will be easier for raphael to pick and choose when integrate batching logic with metric selection.